src/bp/core/routers/bots/index.ts

```diff
@@ -225,6 +225,15 @@ export class BotsRouter extends CustomRouter {
               window.BOTPRESS_FLOW_EDITOR_DISABLED = ${data.flowEditorDisabled};
           `
 
+        const docs = `{
+          flows: 'main/dialog',
+          action: 'main/code',
+          memory: 'main/memory',
+          content: 'main/content',
+          nlu: 'main/nlu',
+          debug: 'main/emulator'
+        }`
```

Comment:
Instead of having this "correspondance table", I suggest using directly the correct link to the documentation instead. It would be easier to find the corresponding page when searching in the code, and would keep the link where it should be. window.DOCS would be unnecessary.



```js
// instead of 
<LinkDocumentationProvider file="memory" />

// use 
<LinkDocumentationProvider file="main/memory" />
```